services/workflows-service/.eslintrc.cjs

```diff
@@ -1,4 +1,5 @@
 const config = require('../../packages/config/eslintrc.base.cjs');
+const checkRepositoryArgsPlugin = require('./plugins/check-repository-args');
```

Comment:
See if we can improve the plugin's name, the plugin is related to `projectId` and multi-tenancy.